### ISSUE TITLE
Questions on enable input in betacode

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios-core#276]
1) while testing issue #184, I noticed that the pop up in the screenshot had the word Mou=sa for Μοῦσα.
if I enter Mo=usa, it shows Μοῦσα as it is suggested in the sample chars.
if I enter Mou=sa, it does not change into Μοῦσα but the output is the same in the popup.
question: should we check for error or is it ok as is?

2) also z = ζ , shouldn't it be z => ζ  ?

**Comment by balmas** [comment]
I'll check.

**Comment by balmas** [comment]
via this issue should also address this:

>q/esqai for θέσθαι -> it worked!!!! I did try few Greek words and it works fine. for πρὸς, shall we add that p is r? or is it redundant?

Yes I think we should add that. All chars that are converted should be listed.

_Originally posted by @balmas in https://github.com/alpheios-project/alpheios-core/issues/104#issuecomment-616559299_

**Comment by balmas** [comment]
re mou=sa, it seems that the code we based this on has reversed the normal order of the betacode accents. The standard is to have the accent marks after the letter and not before.  We need to fix this for all of the letters.

**Comment by balmas** [comment]
we also need to fix use of ! it should be |

**Comment by balmas** [comment]
Ah, I see why they switched the diacritics to before the vowel -- because the javascript to transform the text upon input is much more complex to write if you do it after. I'm going to leave it before.

I have made other changes however, to bring the implementation as close as possible to standard betacode. We should use | instead ! for PROSGEGRAMMENI (unicode 1fbe) and we should convert / into TONOS instead of OXIA (See also note in #239 )

I have also added support for _ for macron and ^ for vrachy and a few other combinations of diacritics that were missing. And I removed support for some non-standard transforms.
